srcpkgs/cockatrice/template

```diff
@@ -0,0 +1,31 @@
+# Template file for 'cockatrice'
+pkgname=cockatrice
+version=2.8.0
+revision=1
+make_use_env=compliant
+hostmakedepends="cmake"
+makedepends="liblzma-devel protobuf protobuf-devel qt5-devel qt5-multimedia-devel qt5-svg-devel qt5-tools-devel qt5-websockets-devel"
+depends="liblzma protobuf qt5 qt5-multimedia qt5-svg qt5-tools qt5-websockets"
+short_desc="Program for playing tabletop card games over a network"
+maintainer="NuclearPine <nuclearpine.h3nbw@aleeas.com>"
+license="GPL-2.0-only"
+homepage="https://cockatrice.github.io"
+distfiles="https://github.com/Cockatrice/Cockatrice/archive/refs/tags/2021-01-26-Release-2.8.0.tar.gz"
+wrksrc="Cockatrice-2021-01-26-Release-2.8.0"
```

Comment:
don't hardcode the version here, use `$version`

you might also want to define a `_reldate` variable for that date, to make things easier to update